src/main/java/org/ladocuploader/app/cli/SubmissionTransfer.java

```diff
@@ -77,6 +76,33 @@ public SubmissionTransfer(SubmissionRepository submissionRepository, Transmissio
 
   @ShellMethod(key = "transferSubmissions")
   public void transferSubmissions() {
+    // Give a 2-hour wait for folks to upload documents
+    OffsetDateTime submittedAtCutoff = OffsetDateTime.now().minusHours(TWO_HOURS);
+    List<Submission> queuedSubmissions = transmissionRepository.submissionsToTransmit(Sort.unsorted(), TransmissionType.SNAP);
+    if (queuedSubmissions.isEmpty()) {
+      log.info("Nothing to transmit");
+      return;
+    }
+    log.info("Found %s queued transmissions".formatted(queuedSubmissions.size()));
+
+    queuedSubmissions = queuedSubmissions.stream()
+        .filter(submission -> (submission.getSubmittedAt().isBefore(submittedAtCutoff))).toList();
+    log.info("Excluding %s submitted within last 2 hours".formatted(queuedSubmissions.size()));
```

Comment:
For this log wouldn't queued submissions size be the number of applications that **are** in the window where they should be sent (as they are older than 2 hours ago?).

Reply:
Oh yeah youre right